corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -56,10 +56,11 @@ hqDefine("cloudcare/js/formplayer/menus/views", [
         },
         attributes: function () {
             const displayText = this.options.model.attributes.displayText;
+            const badgeText = this.options.model.attributes.badgeText;
             return {
                 "role": "link",
                 "tabindex": "0",
-                "aria-label": displayText,
+                "aria-label": displayText + (badgeText ? "; " + badgeText : ""),
```

Comment:
does the ";" have significance?

Reply:
I am actually not sure, if somebody that uses screen readers wants them slowed down on purpose. If I would guess, I would think they prefer otherwise.

Reply:
I did a similar thing with the help text header for required profiles. In that case I added a period to the end of the heading so that there is a natural pause before reading the actual help text. Otherwise the heading and the help text sounded like one long/confusing phrase.